wallet/inputs.go

```diff
@@ -246,10 +246,55 @@ func (lbdar *LinkBraveDepositAccountRequest) HandleErrors(err error) *handlers.A
 	return handlers.ValidationError("brave link wallet request validation errors", issues)
 }
 
+// GeminiLinkingRequest holds info needed to link gemini account
+type GeminiLinkingRequest struct {
+	VerificationToken string `json:"linking_info"`
+}
+
+// Validate - implementation of validatable interface
+func (glr *GeminiLinkingRequest) Validate(ctx context.Context) error {
+	var merr = new(errorutils.MultiError)
+	if glr.VerificationToken == "" {
+		merr.Append(errors.New("failed to validate 'linking_info': must not be empty"))
```

Comment:
why do we need this custom error type when only one is appended? can just use a regular error here.